packages/core/src/tracing/tracingHeaders.js

```diff
@@ -99,6 +105,12 @@ exports.fromHeaders = function fromHeaders(headers) {
   let xInstanaS = readInstanaParentId(headers);
   const levelAndCorrelation = readLevelAndCorrelation(headers);
   const level = levelAndCorrelation.level;
+  // TODO: We will revisit this to implement a more effective design for logging trace level information.
+  logger.debug(
```

Comment:
We will discuss this in the next tracer community meeting to get a common standard for all traces for logging trace level information